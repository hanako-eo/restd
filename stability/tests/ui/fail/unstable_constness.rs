#![feature(staged_api)]
#![stable(feature = "root", since = "1.0.0")]
#![allow(internal_features)]

#[stability::unstable(feature = "test", issue = "none", reason = "sd", constness)]
fn test() {}

fn main() {}
