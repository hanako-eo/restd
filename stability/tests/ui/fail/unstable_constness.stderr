error: attributes `#[rustc_const_unstable]`, `#[rustc_const_stable]` and `#[rustc_const_stable_indirect]` require the function or method to be `const`
 --> tests/ui/fail/unstable_constness.rs:6:1
  |
6 | fn test() {}
  | ^^^^^^^^^
  |
help: make the function or method const
 --> tests/ui/fail/unstable_constness.rs:6:1
  |
6 | fn test() {}
  | ^^^^^^^^^
